raiden/tests/integration/long_running/test_token_networks.py

```diff
@@ -35,8 +35,7 @@ def wait_for_transaction(
 def is_channel_open_and_funded(channel_state):
     return (
         channel.get_status(channel_state) == CHANNEL_STATE_OPENED and
-        channel_state.our_state.contract_balance > 0 and
-        channel_state.partner_state.contract_balance > 0
+        channel_state.our_state.contract_balance > 0
```

Comment:
why did you remove the other test?

Reply:
It required that all channels we have would be counter-deposited, but this may not be the case when we connected to a client already saturated. This will just assert our side of our channels were funded.